### PR TITLE
fix admin commands in dm only

### PR DESCRIPTION
**Pull request by amlcodes**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "wickrio-bot-api",
-  "version": "5.64.9",
+  "version": "5.64.91",
```
Comment (pwcushman) [inline package.json:3]: please change this to 5.64.10

File: src/services/message.js [hunk 2]
```diff
@@ -265,13 +265,11 @@ class MessageService {
...
-      if (this.adminDMonly && this.convoType !== 'personal') {
+      if (this.adminDMonly && convoType !== 'personal') {
         const reply = `Hey admin commands should be sent in direct message to the bot, not in a room`
-        this.wickrAPI.cmdSendRoomMessage(this.messageService.vGroupID, reply)
+        this.wickrAPI.cmdSendRoomMessage(vGroupID, reply)
         return
-      }
-
-      if (!this.adminDMonly) {
+      } else if (this.adminDMonly) {
```
Comment (pwcushman) [inline src/services/message.js:272]: What is the point of an "else if" when the previous if code does a "return" ?

Comment (amlcodes) [reply src/services/message.js:272]: yeah probably redundant

File: src/services/message.js [hunk 3]
```diff
@@ -264,21 +264,18 @@ class MessageService {
     // If this is an admin then process any admin commands
     // if adminDMonly is false
 
-    if (isAdmin) {
-      if (this.adminDMonly && this.convoType !== 'personal') {
+    if (isAdmin && this.adminDMonly && message.includes('/admin')) {
```
Comment (pwcushman) [inline src/services/message.js:267]: This code is wrong.  You are assuming that all admin commands are /admin, this is NOT true.  There are /verify commands as well, which ONLY admins have.

Also, most integrations do NOT use/set the adminDMonly value.

Comment (amlcodes) [reply src/services/message.js:267]: admin dm only is being set as false by default in the api.

**Pull request closed without merging**